src/simulator/simulator.h

```diff
@@ -60,21 +60,22 @@ class Simulator {
   config_reader::ConfigReader reader_;
   config_reader::ConfigReader init_config_reader_;
 
+  // tenporary variables
   Pose2Df vel_;
   Pose2Df cur_loc_;
 
   std::vector<std::unique_ptr<EntityBase>> objects;
 
-  ros::Subscriber initSubscriber;
+  std::vector<ros::Subscriber> initSubscribers;
 
-  ros::Publisher odometryTwistPublisher;
-  ros::Publisher laserPublisher;
-  ros::Publisher viz_laser_publisher_;
+  std::vector<ros::Publisher> odometryTwistPublishers;
```

Comment:
All member variables should end in underscore. 
`std::vector<ros::Publisher> odometryTwistPublishers_`

This is true even if previously existing code did it incorrectly. We are slowly updating the code base to comply with a single style.

Reply:
In compliance to the above comment, I have made the following modification:

```
robotPosMarkers -> robotPosMarkers_
initSubscribers -> initSubscribers_
odometryTwistPublishers -> dometryTwistPublishers_
laserPublishers -> laserPublishers_
mapLinesPublisher -> mapLinesPublisher_
posMarkerPublisher -> posMarkerPublisher_
objectLinesPublisher -> objectLinesPublisher_
truePosePublisher -> truePosePublisher_
localizationPublishers -> localizationPublishers_
br -> br_
scanDataMsg -> scanDataMsg_
odometryTwistMsg -> odometryTwistMsg_
localizationMsg -> ocalizationMsg_
objectLinesMarker -> objectLinesMarker_
truePoseMsg -> truePoseMsg_
```

The only thing that is not changed is `objects`.